### ISSUE TITLE
UnsupportedOperationException in AbstractRule.add()

### ISSUE THREAD

**Issue by hrj** [radkovo/jStyleParser#93]
On latest master branch, commit f4c645b10295ce07f62f76d475ac82a704e414fa

I get the following exception:

```
at java.util.AbstractList.add(AbstractList.java:148)
	at java.util.AbstractList.add(AbstractList.java:108)
	at cz.vutbr.web.csskit.AbstractRule.add(AbstractRule.java:68)
	at cz.vutbr.web.csskit.SelectorImpl.add(SelectorImpl.java:59)
	at cz.vutbr.web.csskit.SelectorImpl.add(SelectorImpl.java:28)
	at cz.vutbr.web.csskit.SelectorImpl$PseudoClassImpl.<init>(SelectorImpl.java:449)
	at cz.vutbr.web.csskit.SelectorImpl$PseudoClassImpl.<init>(SelectorImpl.java:467)
	at cz.vutbr.web.csskit.RuleFactoryImpl.createPseudoClass(RuleFactoryImpl.java:179)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitPseudo(CSSParserVisitorImpl.java:1388)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitSelpart(CSSParserVisitorImpl.java:1271)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitSelector(CSSParserVisitorImpl.java:1236)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitCombined_selector(CSSParserVisitorImpl.java:1162)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitRuleset(CSSParserVisitorImpl.java:696)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitStatement(CSSParserVisitorImpl.java:311)
	at cz.vutbr.web.csskit.antlr4.CSSParserVisitorImpl.visitStylesheet(CSSParserVisitorImpl.java:270)
	at cz.vutbr.web.csskit.antlr4.CSSParserFactory.parse(CSSParserFactory.java:206)
	at cz.vutbr.web.csskit.antlr4.CSSParserFactory.parseAndImport(CSSParserFactory.java:147)
	at cz.vutbr.web.csskit.antlr4.CSSParserFactory.parse(CSSParserFactory.java:72)
	at cz.vutbr.web.csskit.antlr4.CSSParserFactory.parse(CSSParserFactory.java:93)
	at org.lobobrowser.html.style.CSSUtilities.jParseCSS2(CSSUtilities.java:154)
	at org.lobobrowser.html.style.CSSUtilities.jParse(CSSUtilities.java:136)
```

This is while parsing a CSS file from github.com. I hope the stack trace might be enough, else I will try to narrow down the cause.

**Comment by radkovo** [comment]
This should be fixed by 7da8502bb525cd6d5a51ce00214206244f15d578 (but I am not able to test). If it does not help, I will need the corresponding CSS code.

**Comment by hrj** [comment]
Yup it is fixed! Thanks!
